### PR TITLE
Atomic: add 8 plugins to Atomic incompatible plugin list

### PR DESCRIPTION
**Pull request by donpark**
- anywhere-elementor
- anywhere-elementor-pro
- cache-enabler
- fast-velocity-minify
- plugins-garbage-collector
- ultimate-wp-reset
- wp-speed-of-light
- zapp-proxy-server

### CODE REVIEW

**Comment by donpark** [comment]
same result

**Comment by obenland** [comment]
Where does the list of incompatible plugins originate from?

File: client/my-sites/plugins/plugin-meta/index.jsx [hunk 1]
```diff
@@ -306,16 +310,20 @@ export class PluginMeta extends Component {
 			'wp-staging',
 
 			// misc
+			'anywhere-elementor',
```
Comment (retrofox) [inline client/my-sites/plugins/plugin-meta/index.jsx]: ✅ https://wordpress.org/plugins/anywhere-elementor-pro/

File: client/my-sites/plugins/plugin-meta/index.jsx [hunk 2]
```diff
@@ -306,16 +310,20 @@ export class PluginMeta extends Component {
...
+			'anywhere-elementor-pro',
```
Comment (retrofox) [inline client/my-sites/plugins/plugin-meta/index.jsx]: `404` https://wordpress.org/plugins/search/anywhere-elementor-pro/

**Comment by retrofox** [comment]
> Where does the list of incompatible plugins originate from?

https://atomicp2.wordpress.com/2018/09/06/request-to-block-plugins-08-18-2018/

**Comment by retrofox** [comment]
> Where does the list of incompatible plugins originate from?

http://wp.me/p9o2xV-98-p2

**Pull request merged**